website/docs/developer/deployment-guide/testing_deployment.mdx

```diff
@@ -0,0 +1,28 @@
+---
+id: testing_deployment
+title: Testing Deployment
+description: Provides information on testing ORCA code deployment.
+---
+
+import MyImage from '@site/docs/templates/pan-zoom-image.mdx';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The grey fox jumped over the red fern chasing the pink pig.
+
+<MyImage
+    imageSource={useBaseUrl('static/example_test_data/DR.jpg')}
+    imageAlt="A fox chasing a pig."
+    zoomInPic={useBaseUrl('static/img/zoom-in.svg')}
+    zoomOutPic={useBaseUrl('static/img/zoom-out.svg')}
+    resetPic={useBaseUrl('static/img/zoom-pan-reset.svg')}
+/>
+
```

Comment:
Remove this boiler plate since it is not needed.

```
The grey fox jumped over the red fern chasing the pink pig.

<MyImage
    imageSource={useBaseUrl('static/example_test_data/DR.jpg')}
    imageAlt="A fox chasing a pig."
    zoomInPic={useBaseUrl('static/img/zoom-in.svg')}
    zoomOutPic={useBaseUrl('static/img/zoom-out.svg')}
    resetPic={useBaseUrl('static/img/zoom-pan-reset.svg')}
/>
```